### ISSUE TITLE
Advanced SubStation Alpha (ASS) subtitle format support

### ISSUE THREAD

**Issue by kirito41dd** [stashapp/stash#3077]
name.mkv
name.ass

need display .ass subtitle in video

Spec: https://web.archive.org/web/20170625033533/http://www.tcax.org/docs/ass-specs.htm

**Comment by skier233** [comment]
???

**Comment by DogmaDragon** [comment]
> ???

It's an advanced version of the SSA format. <https://en.wikipedia.org/wiki/SubStation_Alpha>

**Comment by ghost** [comment]
> 

No, that's the language of the butt, and you cant change my mind !

**Comment by YurikaL** [comment]
Speaking of ~~ass~~ subtitles: native external players (such as mpv) usually fully support any kind of subtitles that ever exist in known universe, and offloading subtitle rendering (and video playback in general) to such external player would solve this issue automatically! So vote for external remote players support at  #2747!
